src/fhirdefs/load.ts

```diff
@@ -194,7 +194,10 @@ export function loadCustomResources(input: string, defs: FHIRDefinitions): void
       for (const file of files) {
         let resourceJSON: any;
         try {
-          if (file.endsWith('.json')) {
+          if (file.startsWith('.')) {
```

Comment:
We have come across this situation in a different part of our code, and there we used a package called `junk` to filter out hidden files like `.DS_Store` but also certain Windows hidden files like `Thumbs.db`. I think it would be best to be consistent and use that here as well. To use `junk`, you'd want to add:
```
import junk from 'junk';
```
to the top of `load.ts`, and then line 197 would be something like:
```
if (junk.is(file)) {
```
The `junk.is` function just tests if the filename is on `junk`s list of system files.